src/channels/variables-channel.js

```diff
@@ -247,12 +338,22 @@ module.exports = function (options) {
             return id;
         },
 
-
+        /**
+         * Stop receiving notification when a variable is updated. TODO-don't we need the variable names too?
```

Comment:
I'm not sure if I wrote the TODO or you did. Either way, you can remove it, the token already points to some variables (from subscribe)